content_schemas/formats/content_block_postal_address.jsonnet

```diff
@@ -20,6 +20,9 @@
         },
         postcode: {
           type: "string"
+        },
+        content_id_alias: {
+          "$ref": "#/definitions/content_id_alias",
```

Comment:
This feels to be like a metadata element, so should ideally sit outside the `details` object. I _think_ it should be just a case of adding it to the top level, so it looks like this:

```
(import "shared/content_block.jsonnet") + {
  document_type: "content_block_postal_address",
  content_id_alias: {
    "$ref": "#/definitions/content_id_alias",
  },
  definitions: {
    details: {
      type: "object",
      additionalProperties: false,
      required: ["line_1", "town_or_city", "postcode"],
      properties: {
        line_1: {
          type: "string",
        },
        line_2: {
          type: "string",
        },
        town_or_city: {
          type: "string",
        },
        county: {
          type: "string",
        },
        postcode: {
          type: "string"
        },
      },
    },
  },
}
```

Reply:
Thinking about it, that should probably sit in the `shared/content_block.jsonnet` file too

Reply:
I've added a couple of temp commits (for ease of rebase) to demonstrate what I'm thinking.

Basically, I've added a `content_id_alias` field to the [publisher default properties](https://github.com/alphagov/publishing-api/blob/2481672d732cfd01db9af0097308e5275818b275/content_schemas/formats/shared/default_properties/publisher.jsonnet#L35), and [made them forbidden by default](https://github.com/alphagov/publishing-api/blob/2481672d732cfd01db9af0097308e5275818b275/content_schemas/formats/shared/default_format.jsonnet#L14). The Content Block format (which inherits from the default) then [overrides this to make it optional (we might want to make it mandatory later)](https://github.com/alphagov/publishing-api/blob/2481672d732cfd01db9af0097308e5275818b275/content_schemas/formats/shared/content_block.jsonnet#L5)

Hope this makes sense! Any questions, let me know! 👍